.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -350,7 +350,7 @@ jobs:
         uses: ./.github/actions/install-wasi-sdk-wabt
         with:
           os: ${{ matrix.os }}
-
```

Comment:
Maybe keep the empty line

Reply:
The modification added a new line. So, accepting the modification is correct, right?